dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ResourcePolicyRestRepository.java

```diff
@@ -52,9 +95,211 @@ public ResourcePolicyRest findOne(Context context, Integer id) {
         throw new RepositoryMethodNotImplementedException(ResourcePolicyRest.NAME, "findAll");
     }
 
-
     @Override
     public Class<ResourcePolicyRest> getDomainClass() {
         return ResourcePolicyRest.class;
     }
-}
+
+    @PreAuthorize("hasPermission(#resourceUuid, 'dspaceObject', 'ADMIN')")
+    @SearchRestMethod(name = "resource")
+    public Page<ResourcePolicyRest> resource(@Parameter(value = "uuid", required = true) UUID resourceUuid,
+                                      @Parameter(value = "action", required = false) String action, Pageable pageable) {
+
+        List<ResourcePolicy> resourcePolisies = null;
+        int total = 0;
+        try {
+            Context context = obtainContext();
+            if (action != null) {
+                int actionId = Constants.getActionID(action);
+                resourcePolisies = resourcePolicyService.findByResouceUuidAndActionId(context, resourceUuid, actionId,
+                        Math.toIntExact(pageable.getOffset()),
+                        Math.toIntExact(pageable.getOffset() + pageable.getPageSize()));
+                total = resourcePolicyService.countByResouceUuidAndActionId(context, resourceUuid, actionId);
+            } else {
+                resourcePolisies = resourcePolicyService.findByResouceUuid(context, resourceUuid,
+                        Math.toIntExact(pageable.getOffset()),
+                        Math.toIntExact(pageable.getOffset() + pageable.getPageSize()));
+                total = resourcePolicyService.countByResourceUuid(context, resourceUuid);
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+        return converter.toRestPage(resourcePolisies, pageable, total, utils.obtainProjection(true));
+    }
+
+    @PreAuthorize("hasPermission(#epersonUuid, 'EPERSON', 'READ')")
+    @SearchRestMethod(name = "eperson")
+    public Page<ResourcePolicyRest> eperson(@Parameter(value = "uuid", required = true) UUID epersonUuid,
```

Comment:
Again, could you add JavaDocs to this new method? This also looks like it should be named something like `findByEPerson` as it's returning all ResourcePolicies that match a given EPerson.